solutions/pandas_tests/Makefile

```diff
@@ -0,0 +1,76 @@
+# Some tests in files listed in selective_tests fail/ error out on execution.
+# These failures and error are caused by one of below:
+# 1. pypaper linux specific errors, documented
+# here - https://pyperclip.readthedocs.io/en/latest/#not-implemented-error
+# 2. s3fs error (which is currently an optional file handling mechanism in pandas)
+# https://bleepcoder.com/evalml/557744402/unit-test-fails-after-upgrading-to-pandas-1-0-0
+#
+# Make 'run-all' to test the other tests in these files.
+
+TOP = $(abspath ../..)
+include $(TOP)/defs.mak
+
+APPBUILDER=$(TOP)/scripts/appbuilder
+TEST = pandas/tests/arrays/string_/test_string.py
+PACKAGE_PATH = myst/bin/python3
+PYTEST_INI_FILE = pytest.ini
+
+OPTS = --nobrk --max-affinity-cpus=4
+
+ifdef STRACE
+OPTS += --strace
+endif
+
+all: rootfs cpiofs
+
+appdir:
+	$(APPBUILDER) Dockerfile
+	cp $(PYTEST_INI_FILE) appdir/
+
+cpiofs: appdir
```

Comment:
Looks like for running the tests, ext2 rootfs is being used. Are the cpio and package targets required?

Reply:
removed cpio and package targets

Reply:
Probably can simply remove "cpiofs" from line 24 to leave it for development usage and not build it on pipeline run